apis/v1alpha3/vspherefailuredomain_types.go

```diff
@@ -78,6 +78,11 @@ type Topology struct {
 	// virtual machine is created/located.
 	// +optional
 	Datastore string `json:"datastore,omitempty"`
+
+	// StoragePolicy is the name of the policy that is used to target a datastore
+	// in which the virtual machine is created/located
+	// +optional
+	StoragePolicy string `json:"storagePolicy,omitempty"`
```

Comment:
We should only add this to v1beta1.

Reply:
@chrischdi ,
I tried adding the storage policy only in v1beta1, but the conversion-gen fails to generate the `zz_generated.conversion.go` without compilation error due to missing field.
Could you please suggest